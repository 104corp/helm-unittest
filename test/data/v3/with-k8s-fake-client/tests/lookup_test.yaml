templates:
  - templates/lookup.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}
    kubernetes_provider:
      scheme:
        "v1/Pod":
          gvr:
            version:  "v1"
            resource: "pods"
          namespaced: true
      objects:
        - kind: Pod
          apiVersion: v1
          metadata:
            name: exists
            namespace: default

