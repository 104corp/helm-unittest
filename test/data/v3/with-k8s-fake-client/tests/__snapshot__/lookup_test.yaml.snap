manifest should match snapshot:
  1: |
    lookup_exists:
      apiVersion: v1
      kind: Pod
      metadata:
        name: exists
        namespace: default
    lookup_not_exists: {}
